README.md

```diff
@@ -82,6 +104,7 @@ module: {
 - __techs__ <Array>: list of techs extensions for require in runtime, `['js']` by default
 - __techMap__ <Object>: mapping of techs to extensions. Example: `{ 'js' : ['react.js', 'react.ts', 'react.es'], 'css' : ['post.css'] }`
 - __langs__ <Array>: list of langs in which resloves '.i18n' tech
+- __generators__ <Object>: customization of code generators by tech. The function when it is provided receive one argument: __files__ with signature `Array<String>`. This is the list of files of the specified technology, got from current import. Examples: `{ js : null }` or ```{ js: (files) => files.map(file => `require('${file.path}')`).join(',\n') }```
```

Comment:
is it meaningful to do `{ js : null }`?

Reply:
Another option seems to be changing of default "js" generator what require a major release after that. 

The problem is that JS generator adds call of `.applyDecls()` for the module https://github.com/bem/webpack-bem-loader/blob/master/generators/js.js#L11. So the ability to use this loader independently (without https://github.com/bem/bem-react-core) is missed.